bin/counts_summary.Rmd

```diff
@@ -149,7 +149,7 @@ custom_theme <- function(base_size = 12, base_line_size = 1.0, base_rect_size =
 
 ## Total number of mapped reads per sample
 
-```{r, fig.width = figwidth, fig.height = figheight, warning = FALSE}
```

Comment:
sure that `figwidth` and so on is the problem? these parameters are defined earlier in the notebook and set the figure dimensions dynamically depending on sample number (instead of fixed)

Reply:
`figwidth` and so on were not one of the fatal problems, but RStudio claimed the used fig.width & fig.height options were invalid (compare screenshot) - I checked and even though it complains, they work well. I adjusted everything back to `fig.width = figwidth` etc.
![image](https://github.com/MPUSP/nf-core-crispriscreen/assets/83819421/e3925960-ee19-45d1-8b87-9463d2a87e0c)